src/main/java/de/unistuttgart/towercrushbackend/controller/components/WebsocketListener.java

```diff
@@ -1,25 +1,179 @@
 package de.unistuttgart.towercrushbackend.controller.components;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import de.unistuttgart.towercrushbackend.data.websockets.*;
+import de.unistuttgart.towercrushbackend.service.websockets.LobbyManagerService;
+import de.unistuttgart.towercrushbackend.service.websockets.WebsocketService;
 import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.event.EventListener;
+import org.springframework.messaging.simp.SimpMessagingTemplate;
+import org.springframework.messaging.simp.stomp.StompHeaderAccessor;
 import org.springframework.stereotype.Component;
+import org.springframework.web.socket.messaging.SessionConnectEvent;
+import org.springframework.web.socket.messaging.SessionDisconnectEvent;
 import org.springframework.web.socket.messaging.SessionSubscribeEvent;
-import org.springframework.web.socket.messaging.SessionUnsubscribeEvent;
+
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
 
 @Component
 @Slf4j
 public class WebsocketListener {
-    @EventListener(SessionSubscribeEvent.class)
-    public void onSessionSubscribeEvent(SessionSubscribeEvent event) {
-        log.info("OnSessionSubscribeEvent: " + event.toString());
-        log.info("OnSessionSubscribeEvent2 : " + event.getMessage());
-        log.info("OnSessionSubscribeEvent3 : " + event.getUser());
-    }
-
-    @EventListener(SessionUnsubscribeEvent.class)
-    public void onSessionUnsubscribeEvent(SessionUnsubscribeEvent event){
-        log.info("OnSessionSubscribeEvent: " + event.toString());
-        log.info("OnSessionSubscribeEvent2 : " + event.getMessage());
-        log.info("OnSessionSubscribeEvent3 : " + event.getUser());
+
+    @Autowired
+    LobbyManagerService lobbyManagerService;
+    @Autowired
+    WebsocketService websocketService;
+    @Autowired
+    SimpMessagingTemplate simpMessagingTemplate;
+    ExecutorService executorService =
+        Executors.newFixedThreadPool(1);
+    final String lobbyTopic = "/topic/lobby/";
+    final String developerTopic = "/topic/developer/";
```

Comment:
```suggestion
    static final String LOBBY_TOPIC = "/topic/lobby/";
    static final String DEVELOPER_TOPIC = "/topic/developer/";
```
Naming convention for constants in Java

Reply:
sure